RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -951,16 +924,16 @@ void lst::Event<Acc3D>::createPixelQuintuplets() {
   auto* connectedPixelSize_host = connectedPixelSize_host_buf.data();
   auto* connectedPixelIndex_host = connectedPixelIndex_host_buf.data();
 
-  int pixelIndexOffsetPos =
-      pixelMapping_.connectedPixelsIndex[size_superbins - 1] + pixelMapping_.connectedPixelsSizes[size_superbins - 1];
-  int pixelIndexOffsetNeg = pixelMapping_.connectedPixelsIndexPos[size_superbins - 1] +
-                            pixelMapping_.connectedPixelsSizesPos[size_superbins - 1] + pixelIndexOffsetPos;
+  int pixelIndexOffsetPos = pixelMapping_.connectedPixelsIndex[::size_superbins - 1] +
+                            pixelMapping_.connectedPixelsSizes[::size_superbins - 1];
+  int pixelIndexOffsetNeg = pixelMapping_.connectedPixelsIndexPos[::size_superbins - 1] +
+                            pixelMapping_.connectedPixelsSizesPos[::size_superbins - 1] + pixelIndexOffsetPos;
```

Comment:
is this still needed considering `    using ::lst::size_superbins;` in RecoTracker/LSTCore/interface/alpaka/Constants.h ?